glib/Markup.cs

```diff
@@ -30,15 +30,16 @@ public class Markup {
 		private Markup () {}
 		
 		[DllImport("libglib-2.0-0.dll", CallingConvention=CallingConvention.Cdecl)]
-		static extern IntPtr g_markup_escape_text (IntPtr text, int len);
+		static extern IntPtr g_markup_escape_text (IntPtr text, IntPtr len);
```

Comment:
Will this break on 64bit windows? Do we care?

Reply:
It's a C long?

Reply:
Exactly - https://docs.microsoft.com/en-us/cpp/build/common-visual-cpp-64-bit-migration-issues?view=vs-2019

Not sure if win64 is a supported platform or not. If it is then we have a problem with all longs. If it's not then do we bail out during init, or corrupt memory at runtime?

Reply:
OH GOD.

Reply:
I'll replace with a delegate which chooses at runtime which one is used.

Reply:
But actually, win64 is not supported. I will revert the patch.